src/sdl2_compat.c

```diff
@@ -5813,6 +5836,8 @@ SDL_InitSubSystem(Uint32 flags)
         return -1;
     }
 
+    SynchronizeEnvironmentVariables();
```

Comment:
I don't know if it's a big deal, but this is going to synchronize multiple times if flags is SDL_INIT_VIDEO or SDL_INIT_AUDIO